pkg/api/repository_rpm.go

```diff
@@ -18,8 +18,9 @@ type RepositoryRpmCollectionResponse struct {
 }
 
 type SearchRpmRequest struct {
-	URLs   []string `json:"urls"`   // URLs of repositories to search
-	Search string   `json:"search"` //Search string to search rpm names
+	URLs   []string `json:"urls,omitempty"`  // URLs of repositories to search
+	Uuids  []string `json:"uuids,omitempty"` // List of RepositoryConfig uuids to serach
```

Comment:
```suggestion
	Uuids  []string `json:"uuids,omitempty"` // List of RepositoryConfig UUIDs to search
```

Reply:
Just for awareness, this change only needs to be made here. The other files where this typo is were auto-generated using this file.